.github/workflows/elixir.yml

```diff
@@ -6,7 +6,6 @@ permissions:
 on:
   workflow_dispatch:
   merge_group:
-    branches: [develop]
```

Comment:
We want to target the temporary branch that a rebase will be made to ensure that after rebase CI passes before merge